tox.ini

```diff
@@ -1,7 +1,18 @@
 [tox]
-envlist = py27,py35,py36,py37,py38
+envlist = py{27,35,36,37,38}-{default,use_chardet_on_py3}
 
 [testenv]
-
+deps = -rrequirements-dev.txt
+extras =
+    security
+    socks
 commands =
-    python setup.py test
+    pytest tests
+
+[testenv:default]
+
+[testenv:use_chardet_on_py3]
+extras =
+    security
+    socks
+    lgpl
```

Comment:
@potiuk missed a « lgpl » here. It is the last one.

Reply:
Too much reliance on Ci (which is not running because of workflow awaiting approval :)

Reply:
This time I actually run it before last fixup locally.